api-report/test-runtime-utils.api.md

```diff
@@ -141,7 +141,7 @@ export class MockContainerRuntimeFactory {
     minSeq: Map<string, number>;
     // (undocumented)
     get outstandingMessageCount(): number;
-    processAllMessages(): void;
+    processAllMessages(clients?: string[]): void;
```

Comment:
with my other idea, we could also consider pushing the op array into the mock runtimes, and supporting process directly on those, rather than needing to map them